server/services/user.js

```diff
@@ -10,10 +10,15 @@ const DEVICE_SCHEMA = Joi.object().keys({
   contactInformation: Joi.string()
 });
 
+function exists(userId) {
+  return User.exists(userId);
+}
+
 function createUser(userObject) {
   const userSchema = Joi.object().keys({
     name: Joi.string().required(),
-    email: Joi.string().email().required()
+    email: Joi.string().email().required(),
+    isSysAdmin: Joi.boolean()
```

Comment:
Missed that, yeah it's still behind validation which doesn't allow for the isSysAdmin field to be passed in through the api